common.go

```diff
@@ -2,36 +2,15 @@ package moviebuff
 
 import (
 	"net/http"
-	"os"
 )
 
-var (
-	hostUrl = "https://api.moviebuff.com/api/v2"
-)
-
-func init() {
-	if os.Getenv("MOVIEBUFF_URL") != "" {
-		hostUrl = os.Getenv("MOVIEBUFF_URL")
-	}
-}
-
-type logger interface {
-	Println(v ...interface{})
-}
-
-// Dummy discard, satisfies io.Writer without importing io or os.
-type devNull struct{}
-
-func (devNull) Write(p []byte) (int, error) {
-	return len(p), nil
-}
+func prepareRequest(hostURL, staticToken, path string) (*http.Request, error) {
 
-func prepareRequest(token, path string) (*http.Request, error) {
-	r, err := http.NewRequest(http.MethodGet, hostUrl+path, nil)
+	r, err := http.NewRequest(http.MethodGet, hostURL+path, nil)
 	if err != nil {
 		return nil, err
 	}
 
-	r.Header.Add("X-Api-Key", token)
+	r.Header.Add("X-Api-Key", staticToken)
```

Comment:
Can make `X-Api-Key` constant. Also, why are we adding `staticToken` to it? Just curious.

Reply:
Changed to `const`, we do this for CORS.

Reply:
I feel this is better option to handle CORS with the help of native package `net/http`, we don't need to depend on third party package for this simple header settings. What do you feel about it?